autopush/log_check.py

```diff
@@ -0,0 +1,25 @@
+import uuid
+
+import cyclone.web
+
+from autopush.endpoint import AutoendpointHandler
+
+
+class LogCheckHandler(AutoendpointHandler):
+
+    def initialize(self, ap_settings):
+        self.ap_settings = ap_settings
+        self.request_id = str(uuid.uuid4())
+        self._client_info = self._init_info()
+
+    @cyclone.web.asynchronous
+    def get(self):
+        """HTTP GET
+
+        Generate a dummy error message for logging
+
+        """
+        self.log.error(format="Test Error Message",
+                       status_code=418, errno=0,
+                       **self._client_info)
```

Comment:
This should throw an exception, then report it via log.failure so we can verify in dev/stage that Sentry gets it.